core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -244,15 +266,14 @@ object Battle {
                         else " [" + defender.getName() + "]"
                     else " our [" + defender.getName() + "]"
             val notificationString = attackerString + whatHappenedString + defenderString
-            val cityIcon = "ImprovementIcons/Citadel"
-            val attackerIcon = if (attacker is CityCombatant) cityIcon else attacker.getName()
-            val defenderIcon = if (defender is CityCombatant) cityIcon else defender.getName()
+            val attackerIcon = if (attacker is CityCombatant) NotificationIcon.City else attacker.getName()
+            val defenderIcon = if (defender is CityCombatant) NotificationIcon.City else defender.getName()
```

Comment:
Would it be more clean if there was an `getIconName()` function on the `ICombatant` interface?

Reply:
The idea came to me, too, but hey, gotta stop somewhere.